github/ci/prow/files/jobs/kubevirt/kubevirt-csi/kubevirt-csi-postsubmits.yaml

```diff
@@ -0,0 +1,51 @@
+postsubmits:
+  kubevirt/kubevirt-csi:
+    - name: publish-kubevirt-csi
+      branches:
+        - master
+      decorate: true
+      max_concurrency: 1
+      extra_refs:
+        - org: kubevirt
+          repo: project-infra
+          base_ref: master
+      labels:
+        preset-dind-enabled: "true"
+        preset-docker-mirror: "true"
+        preset-kubevirtci-quay-credential: "true"
+      spec:
+        nodeSelector:
+          type: bare-metal-external
+        volumes:
+          - name: token
+            secret:
+              secretName: oauth-token
+          - name: gcs
+            secret:
+              secretName: gcs
+        containers:
+          - image: gcr.io/k8s-testimages/kubekins-e2e@sha256:8c80c98d035a0f285ad49b964e5d0b62004844f3340407367841618e80e1503c
+            env:
+              - name: GIT_ASKPASS
+                value: "../project-infra/hack/git-askpass.sh"
+              - name: GOOGLE_APPLICATION_CREDENTIALS
+                value: /etc/gcs/service-account.json
+            command:
+              - "/usr/local/bin/runner.sh"
+              - "/bin/bash"
+              - "-c"
+              - >
+                cat $DOCKER_PASSWORD | docker login --username $(<$DOCKER_USER) --password-stdin &&
```

Comment:
You need to login to quay like this IMHO: https://github.com/kubevirt/project-infra/blob/84fe10c981529d026bd5779a1f1f3123c139fa32/github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.34.yaml#L386